<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-27T10:26:25 EDT" name="Default suite" finished-at="2022-08-27T10:26:37 EDT" duration-ms="12208">
    <groups>
    </groups>
    <test started-at="2022-08-27T10:26:25 EDT" name="Default test" finished-at="2022-08-27T10:26:37 EDT" duration-ms="12208">
      <class name="variouspackage.testng">
        <test-method is-config="true" signature="readingconfig()[pri:0, instance:variouspackage.testng@3d5c822d]" started-at="2022-08-27T10:26:25 EDT" name="readingconfig" finished-at="2022-08-27T10:26:25 EDT" duration-ms="15" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- readingconfig -->
        <test-method is-config="true" signature="init()[pri:0, instance:variouspackage.testng@3d5c822d]" started-at="2022-08-27T10:26:25 EDT" name="init" finished-at="2022-08-27T10:26:33 EDT" duration-ms="8739" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method signature="logintest()[pri:0, instance:variouspackage.testng@3d5c822d]" started-at="2022-08-27T10:26:33 EDT" name="logintest" finished-at="2022-08-27T10:26:37 EDT" duration-ms="3381" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[page not found expected [Dashboard] but found [[[FirefoxDriver: firefox on WINDOWS (8a02d9f8-2307-4c74-8a05-0b5a9ffd6e01)] -> xpath: //h2[@style='color: #2F4050; font-size: 16px; font-weight: 400; margin-top: 18px']]]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: page not found expected [Dashboard] but found [[[FirefoxDriver: firefox on WINDOWS (8a02d9f8-2307-4c74-8a05-0b5a9ffd6e01)] -> xpath: //h2[@style='color: #2F4050; font-size: 16px; font-weight: 400; margin-top: 18px']]]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at variouspackage.testng.logintest(testng.java:88)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintest -->
      </class> <!-- variouspackage.testng -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
